packages/decode/package.json

```diff
@@ -34,11 +34,11 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@hopae/sd-jwt-node-crypto": "2.0.3"
+    "@sd-jwt/crypto-nodejs": "workspace:*"
   },
   "dependencies": {
-    "@hopae/sd-jwt-type": "2.0.3",
-    "@hopae/sd-jwt-util": "2.0.3"
+    "@sd-jwt/types": "workspace:*",
```

Comment:
@cre8 This part has actual version. So I changed it to workspace:* again.
I think lerna publish set the version here. Was this intentional?

Reply:
Yeah `workspace:*` should be swapped by lerna for the release. You can check the setup at https://github.com/berendsliedrecht/sd-jwt-ts as we used the exact same setup there.